blue_hardware_drivers/src/BLDCDriver_communication_freq_test.cpp

```diff
@@ -105,12 +85,45 @@ int main(int argc, char **argv) {
       }
     }
   }
+}
 
+int main(int argc, char **argv) {
+  ros::init(argc, argv, "comms", ros::init_options::AnonymousName);
+
+  ros::NodeHandle n;
+  std::vector<comm_id_t> board_list;
+  std::map<comm_id_t, std::vector<double> > velocity_history_mapping;
+
+
+  char* port = argv[1];
+  char* board_nums = argv[2];
+  char* enumerate = argv[3];
+
+  std::string num = "";
+  for (unsigned int i = 0; board_nums[i] != '\0'; i++) {
+    if (board_nums[i] == ',')
+      continue;
+    comm_id_t id_num = std::atoi(board_nums + i);
```

Comment:
nit: may be worth explicitly marking `const` variables as `const`; also applies to char pointers above